cat/bsdcat.c

```diff
@@ -126,16 +126,17 @@ main(int argc, char **argv)
 	bsdcat->argv = argv;
 	bsdcat->argc = argc;
 
-	while ((c = bsdcat_getopt(bsdcat)) != -1) {
+	if ((c = bsdcat_getopt(bsdcat)) != -1) {
 		switch (c) {
 		case 'h':
 			usage(stdout, 0);
-			break;
+			/* NOTREACHED */
```

Comment:
I'm not sure we should remove the `break` here.  It seems that could cause problems for any platforms where `__LA_NORETURN` is not defined.

Reply:
In particular, the tests are currently failing on Ubuntu because
```
/home/runner/work/libarchive/libarchive/cat/bsdcat.c:132:25: error: this statement may fall through [-Werror=implicit-fallthrough=]
  132 |                         usage(stdout, 0);
      |                         ^~~~~~~~~~~~~~~~
```